min-image/runtimes/python/server.py

```diff
@@ -177,15 +185,16 @@ def main():
 
     global bootstrap_path
 
-    if len(sys.argv) < 2:
-        print("Expected execution: chroot <path_to_root_fs> python3 server.py <path_to_bootstrap.py> [cgroup-count] [enable-seccomp]")
+    if len(sys.argv) < 3:
```

Comment:
Let's add a TODO to move to named args (in a future PR) now that we're getting more than a couple.